packages/sanity/src/core/comments/context/comments/CommentsProvider.tsx

```diff
@@ -60,9 +65,50 @@ type TransactionId = string
 /**
  * @beta
  */
-export const CommentsProvider = memo(function CommentsProvider(props: CommentsProviderProps) {
-  const {children, documentId, documentType, isCommentsOpen, onCommentsOpen, sortOrder, type} =
-    props
+export function CommentsProvider(props: CommentsProviderProps) {
+  const parentContext = useContext(CommentsContext)
```

Comment:
I'm not sure I understand why this is needed. Could you clarify in what situations we might end up with nested comment providers?

Reply:
I don't think I understand either lol. I think I was confused. I removed it and put it closer to how it was 😇